R/plumber.R

```diff
@@ -493,7 +493,13 @@ plumber <- R6Class(
 
 #' @rdname plumber
 #' @export
-plumb <- function(file){
+plumb <- function(file, dir){
+  if(!xor(missing(file), missing(dir))){
+    stop("plumber needs only one of a file or a directory with a `plumber.R` file in its root.")
+  } else if (missing(file)){
+    stri_trim_right(dir, pattern = "[/]")
```

Comment:
There's no way you could have known this, but I'm hoping to eliminate the stringi dependency in the future. Can we use R primitives for this instead?

I think:

```
dir <- sub("/$", "", dir)
```